doc/README.md

```diff
@@ -0,0 +1,13 @@
+All of the SunPy documentation is contained in the doc/source folder and code
+comments. To generate the documentation you must have Sphinx (as well as
+Numpydoc and astropy-helpers) installed on your computer. Enter
+the doc/source folder and run:
+
+  make html
```

Comment:
we should probably be selling the `python setup.py build_sphinx` method of doing this.